scholarly/publication.py

```diff
@@ -141,11 +145,11 @@ def fill(self):
                 elif key == 'Publisher':
                     self.bib['publisher'] = val.text
                 elif key == 'Publication date':
-                    self.bib['year'] = arrow.get(val.text).year
+                    self.bib['year'] = arrow.get(val.text, ['YYYY/M']).year
```

Comment:
Let's use the fix recommended by the reporter of the ticket:

 arrow.get(val.text, ['YYYY/M','YYYY/MM/DD', 'YYYY', 'YYYY/M/DD', 'YYYY/M/D', 'YYYY/MM/D']).year